content/docs/examples/bookinfo/index.md

```diff
@@ -66,42 +66,42 @@ To start the application, follow the instructions corresponding to your Istio ru
 If you use GKE, please ensure your cluster has at least 4 standard GKE nodes. If you use Minikube, please ensure you have at least 4GB RAM.
 {{< /tip >}}
 
-1. Change directory to the root of the Istio installation directory.
+1.  Change directory to the root of the Istio installation.
```

Comment:
```suggestion
1.  Go to the root directory of the Istio installation.
```

Reply:
Nack, change directory is the correct operation.